k8s/prod/common/divorce/div-hm.yaml

```diff
@@ -27,7 +27,7 @@ spec:
       readinessPeriod: 15
       useInterpodAntiAffinity: true
       applicationInsightsInstrumentKey: "baaed5d0-7666-4dce-bf74-edfceeb566f9"
-      image: hmctspublic.azurecr.io/div/hm:prod-468cf5ac
+      image: hmctspublic.azurecr.io/div/hm:prod-c1804250
```

Comment:
let's make it work on AAT first before we add prod

Reply:
👍